ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -615,6 +622,10 @@ def test_frontend_function(
             **kwargs_for_test,
         )
 
+        assert ivy.nested_map(
+            ret, lambda x: _is_frontend_array(x) if ivy.is_array(x) else True
```

Comment:
`ivy.is_array()` would be bound to global ivy, which has it's backend unchanged, we should use local Ivy.